### PR TITLE
RND-1002 Remove LiveEditDecorator, LiveEdit and test modernization of compontent stories

### PR DESCRIPTION
**Pull request by PatrycjaKubiczek**
## Description

- remove LiveEditDecorator
- remove LiveEdit 
- after removing these I need to update main.ts because I was getting errors:
```
ERR! /home/patrycjakubiczek/repos/cloudify-ui-components/.storybook/main.ts:1
ERR! import _ from 'lodash';
ERR! ^^^^^^
ERR! 
ERR! SyntaxError: Cannot use import statement outside a module
ERR!     at Object.compileFunction (node:vm:352:18)
ERR!     at wrapSafe (node:internal/modules/cjs/loader:1033:15)
ERR!     at Module._compile (node:internal/modules/cjs/loader:1069:27)
ERR!     at Module._compile (/home/patrycjakubiczek/repos/cloudify-ui-components/node_modules/pirates/lib/index.js:136:24)
ERR!     at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
ERR!     at Object.newLoader [as .ts] (/home/patrycjakubiczek/repos/cloudify-ui-components/node_modules/pirates/lib/index.js:141:7)
ERR!     at Module.load (node:internal/modules/cjs/loader:981:32)
ERR!     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
ERR!     at Module.require (node:internal/modules/cjs/loader:1005:19)
ERR!     at require (node:internal/modules/cjs/helpers:102:18)
ERR!  /home/patrycjakubiczek/repos/cloudify-ui-components/.storybook/main.ts:1
ERR! import _ from 'lodash';
ERR! ^^^^^^
ERR! 
ERR! SyntaxError: Cannot use import statement outside a module
ERR!     at Object.compileFunction (node:vm:352:18)
ERR!     at wrapSafe (node:internal/modules/cjs/loader:1033:15)
ERR!     at Module._compile (node:internal/modules/cjs/loader:1069:27)
ERR!     at Module._compile (/home/patrycjakubiczek/repos/cloudify-ui-components/node_modules/pirates/lib/index.js:136:24)
ERR!     at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
ERR!     at Object.newLoader [as .ts] (/home/patrycjakubiczek/repos/cloudify-ui-components/node_modules/pirates/lib/index.js:141:7)
ERR!     at Module.load (node:internal/modules/cjs/loader:981:32)
ERR!     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
ERR!     at Module.require (node:internal/modules/cjs/loader:1005:19)
ERR!     at require (node:internal/modules/cjs/helpers:102:18)

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.

/home/patrycjakubiczek/repos/cloudify-ui-components/.storybook/main.ts:1
import _ from 'lodash';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1033:15)
    at Module._compile (node:internal/modules/cjs/loader:1069:27)
    at Module._compile (/home/patrycjakubiczek/repos/cloudify-ui-components/node_modules/pirates/lib/index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Object.newLoader [as .ts] (/home/patrycjakubiczek/repos/cloudify-ui-components/node_modules/pirates/lib/index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)`
```
it would be good to investigate it however right now to temporary solve the issue I replace `import` with `require()`
- update IDS and ApproveButton as an example with args
- update a couple of loki img

I waited with "Update IDS React components guidelines (README.md file in IDS components folder)" - because I would like a discussion about how the components in storybooks look like because it could be achieved in different ways

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

CI checks are green ✅ 

## Documentation

N/A

### CODE REVIEW

File: src/components/ids/IdsButton/IdsButton.stories.tsx [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+    component: IdsButton
+} as ComponentMeta<typeof IdsButton>;
 
 type IdsButtonStory = ComponentStory<typeof IdsButton>;
 
-export const basic: IdsButtonStory = () => <IdsButton kind="primary">Primary button</IdsButton>;
+const template: ComponentStory<typeof IdsButton> = args => <IdsButton {...args}>Primary button</IdsButton>;
+export const Basic = template.bind({});
+Basic.args = {
+    kind: 'primary'
```
Comment (qooban) [inline src/components/ids/IdsButton/IdsButton.stories.tsx:16]: Check https://docs.cloudify.co/ui-components/2.42.3/?path=/docs/ids-idsbutton--basic - there's more custom props that we can pass here.

On the other hand - I'm not sure you have to do that within RND-1002, maybe it should be part of [RND-1076](https://cloudifysource.atlassian.net/browse/RND-1076).

File: .storybook/main.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
-import _ from 'lodash';
-import path from 'path';
+/* eslint-disable @typescript-eslint/no-var-requires */
+const _ = require('lodash');
+const path = require('path');
 
-export default {
+module.exports = {
```
Comment (qooban) [inline .storybook/main.ts:5]: Saw the reason of that in PR description and did a quick check. 

Strange that it popped up right now, maybe some dependencies (package lock file) changed after removing live code editor? 🤔 

Adjusting `babel.config.js` to use `'@babel/preset-env'` preset always (not in certain conditions as it is right now) solves it, but I'm not sure it's the right way to go. I don't know if other places won't be affected, so... Unless @Vorbert-Kruk has any other idea I would leave it.

**Review by qooban — COMMENTED** [review]
Added 2 comments.

> I waited with "Update IDS React components guidelines (README.md file in IDS components folder)" - because I would like a discussion about how the components in storybooks look like because it could be achieved in different ways

OK, let's have a talk.

File: .storybook/main.ts (hunk 2 above)
Comment (Vorbert-Kruk) [reply .storybook/main.ts:5]: Unfortunately I don't have any other ideas

But from what I can add, I've been investigating this subject on Friday and it appears that `surcrose` package (which is a dependency of `@homemx/react-live`) was enabling us to use CommonJS syntax
It's quite surprising - I wish that I could explain the background of it, but I didn't want to spend that much time on the investigation

**Review by Vorbert-Kruk — DISMISSED** [review]

File: .storybook/main.ts (hunk 2 above)
Comment (PatrycjaKubiczek) [reply .storybook/main.ts:5]: @qooban @Vorbert-Kruk if I modify babel.config.js and remove [`L26` ](https://github.com/cloudify-cosmo/cloudify-ui-components/blob/master/babel.config.js#L26) I could change require to imports and storybook build without issues 🤔 I'm not sure why we have different setting for testEnv, so I'd appreciate knowledge sharing :)

File: src/components/ids/IdsButton/IdsButton.stories.tsx (hunk 1 above)
Comment (PatrycjaKubiczek) [reply src/components/ids/IdsButton/IdsButton.stories.tsx:16]: even though I don't explicitly describe args here they are still passed in storybook 
Screenshot 2023-08-07 145834

I did it as an example to test if's working we can expand it as part of RND-1076

File: .storybook/main.ts (hunk 2 above)
Comment (qooban) [reply .storybook/main.ts]: As I mentioned in the first comment I think it's not good to change that without understanding the consequences. Looking at Git history it seems that it was intentionally set only for test environment (when `NODE_ENV` is set to `test` according to https://babeljs.io/docs/config-files#apienv) as output package source code (files in `es` folder created after build) was not meant to be transpiled.

You can play with config adjustments (apply `@babel/preset-env` not only in test environment, but also always on `.storybook` folder), but I'm not sure it's worth right now, I would keep the original change without babel config modifications.

File: src/components/ids/IdsButton/IdsButton.stories.tsx (hunk 1 above)
Comment (qooban) [reply src/components/ids/IdsButton/IdsButton.stories.tsx:16]: Great, good to know. In that case, let's leave it for RND-1076.

**Review by qooban — COMMENTED** [review]
Commented.

File: .storybook/main.ts (hunk 2 above)
Comment (PatrycjaKubiczek) [reply .storybook/main.ts:5]: okay, reverted last commit: https://github.com/cloudify-cosmo/cloudify-ui-components/pull/271/commits/d8ad442271deb92cf46faf4f66fdb60a5c196557 👍

**Pull request merged**